Update documentation about LZO and LZ4 modules (#85)
fox-it/dissect-docs: docs/source/projects/dissect.squashfs/index.rst

    $ pip install dissect.squashfs

> This project decompresses LZO and LZ4 compressed file systems and can use a faster, native (Rust-based) LZO or LZ4 implementation,
> instead of a slower pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`.

Reviewer: ```suggestion
This project decompresses LZO and LZ4 compressed file streams. For this it uses native rust-based LZO or LZ4 implementations, instead of their slower pure Python implementations. Both implementations are provided by :doc:`/projects/dissect.util/index`.
```

Author: ```suggestion
This project supports file systems with LZO and LZ4 compression. It can use either a fast Rust-based implementation, or a slow pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`, and the faster implementation will automatically be used if available.
Pre-build wheels are available for most common platforms. In the rare case that a pre-build wheel is not available, please refer to :doc:`/projects/dissect.util/index` for build instructions.
```
